src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkConfig.java

```diff
@@ -218,4 +220,31 @@ private void validateClusteringOrder(@Nullable Schema schema) {
       }
     }
   }
+
+  private void validateOperationProperties(@Nullable Schema schema) {
+    if (Arrays.stream(Operation.values()).map(Enum::name).noneMatch(operation.toUpperCase()::equals)) {
+      throw new InvalidConfigPropertyException(String.format("'%s' is incorrect value for field 'Operation'. " +
+                                                               "This field should contain one of the next values: " +
+                                                               "INSERT, UPDATE or UPSERT.", operation), NAME_OPERATION);
```

Comment:
these show up to the user as 'Insert', 'Update', or 'Upsert', so we should use that instead of all-caps.

Reply:
We can show with Uppercase as first letter and then still pass All CAPS if a drop down is used.

Reply:
Done.